### DISCUSSION TITLE
How to add combobox to grid programmatically?

### DISCUSSION THREAD

**Discussion by vldmrklnk** [AvaloniaUI/Avalonia#9545] [Q&A] [answered]
Hi, guys! The main goal is to after button "Add" was clicked, the new row should added to the grid. This row should have "ComboBox" in every column.

**Comment by maxkatz6** [comment, marked as answer]
Something like this
```
var comboBox = new ComboBox();
comboBox[Grid.RowProperty] = 1:
MyGrid.Children.Add(comboBox);
```

**Reply by vldmrklnk** [reply]
Thank you a lot! I was trying to do it another way at all. I didn't think about grid properties in ComboBox. I was trying to take last cell in grid and paste ComboBox to it.
